input/_Navbar.cshtml

```diff
@@ -9,9 +9,9 @@
         Tuple.Create("FAQ", Context.GetLink("faq")),
         Tuple.Create("<i class=\"fa fa-github\"></i> Source", "https://github.com/cake-build/cake")
     };
-    foreach(Tuple<string, string> page in pages)
+    foreach(Tuple<string, string> p in pages)
```

Comment:
Good, glad you figured this out - the need to use something other than `page` as a variable name to avoid `@page` syntax since that now has special meaning is basically the biggest breaking in 2.x

Reply:
Yip, migrated a few sites now to 2.x, so knew about this one ;)